packages/grafana-ui/src/components/Button/styles.ts

```diff
@@ -131,10 +125,5 @@ export const getButtonStyles = stylesFactory(({ theme, size, variant, withIcon }
       display: flex;
       align-items: center;
     `,
-    icon: css`
-      label: button-icon;
-      margin-right: ${iconDistance};
```

Comment:
So using &nbsp; space in the content instead of this. The spacing is pretty similar and is more relative to font size rather than needed to be manually synced.